src/System.Sorting/tests/SortingTest.cs

```diff
@@ -0,0 +1,84 @@
+            int[] array = new int[6000000];
+            int[] arrayTry = new int[2500];
+            for (int i = 0; i < 2500; i++)
+            {
+                arrayTry[i] = i;
+            }
+            for (int j = 0; j < 2400; j++)
+            {               
+                arrayTry.CopyTo(array, (j * 2500));
+            }
+           
+            int[] arrayClone = (int[])(array.Clone());
+
+
+            TimSort<int> timsort = new TimSort<int>(array, iComparer);
```

Comment:
It would be good to have performance tests that measure different sizes of arrays instead of testing just this one case.

In addition, tests where the array is partially sorted, completely sorted, and random would be good to add.  That will give us a more complete picture of the overall performance of the feature.